app/v1/services/PenaltiesService.scala

```diff
@@ -35,4 +35,12 @@ class PenaltiesService @Inject()(connector: PenaltiesConnector) extends Logging
                         correlationId: String): Future[ServiceOutcome[PenaltiesResponse]] = {
     connector.retrievePenalties(request)
   }
+
```

Comment:
these will need seperating out as the error handeling is going to need reworking to work like the others to account for the fact that theres morre than just 1 error for each error code. cant return the same 400 for all errors

Reply:
the error handing is handled by the Implicit Errorhandler ill send you the link